app/helpers/application_helper.rb

```diff
@@ -1,2 +1,20 @@
 module ApplicationHelper
+  def current_year
+    current_year = Time.now.year
```

Comment:
```suggestion
    Time.now.year
```
но и тут есть проблемка:
в рельсах рекомендуется использовать `Time.current` вместо `Time.now` (Time.new) и `Date.current` вместо `Date.today`
`.current` возвращает текущий момент времени с учетом настройки часового пояса в приложении